libs/clients/driving-license-book/src/lib/drivingLicenseBookClient.service.ts

```diff
@@ -209,7 +210,9 @@ export class DrivingLicenseBookClientApiFactory {
     const activeBook = await this.getActiveBookId(nationalId)
 
     const book = data?.books?.filter((b) => b.id === activeBook && !!b.id)[0]
-
+    const hasPracticeDriving = activeBook
+      ? await this.hasPracticeDriving(activeBook)
+      : false
```

Comment:
```suggestion
    const hasPracticeDriving = activeBook && await this.hasPracticeDriving(activeBook)
```

Reply:
This can yield null, or undefined so sticking with the safer option